packages/gatsby/src/schema/schema.js

```diff
@@ -785,12 +789,34 @@ const addThirdPartySchemas = ({
         type.name !== `Date` &&
         type.name !== `JSON`
       ) {
+        const isCompositeType =
```

Comment:
NIT: can we rename it to 
```suggestion
        const typeHasFields =
```
or something like that?

There is `GraphQLCompositeType` and some utils for it, which means something else so I'd like to avoid using "composite" description here that differs from what `graphql-js` means by it ( ref https://github.com/graphql/graphql-js/blob/00ef21739731d1ea7fc61e4294b5bec5a2e535f2/src/type/definition.js#L289-L308 )

Reply:
Oh, yeah. I forgot that union is also considered to be a `composite` type.